Source/WebKit/UIProcess/API/Cocoa/WKWebViewConfiguration.mm

```diff
@@ -179,6 +179,8 @@ @implementation WKWebViewConfiguration {
     double _sampledPageTopColorMaxDifference;
     double _sampledPageTopColorMinHeight;
     BOOL _allowsInlinePredictions;
+    BOOL _scrollToTextFragmentIndicatorEnabled;
+    BOOL _scrollToTextFragmentMarkingEnabled;
```

Comment:
There's an ongoing effort to move ivars from `WKWebViewConfiguration` to `API::PageConfiguration`, so we should do in this PR before merging.

See https://github.com/WebKit/WebKit/pull/26157 for examples.